modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -21,15 +21,25 @@ Doing so has no impact on how or for how long Prometheus stores metrics.
 * You have installed the OpenShift CLI (`oc`).
 * You have set up a remote write compatible endpoint (such as Thanos) and know the endpoint URL.
 See the link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Prometheus remote endpoints and storage documentation] for information about endpoints that are compatible with the remote write feature.
-* You have set up authentication credentials for the remote write endpoint. 
+* You have set up authentication credentials in a `Secret` object for the remote write endpoint.
+The secret needs to be created in the same namespace as the Prometheus object for which you configure remote write: `openshift-monitoring` for default platform monitoring or `openshift-user-workload-monitoring` for user workload monitoring.
```

Comment:
```suggestion
["You must create the secret" OR "The secret must be"] in the same namespace as the Prometheus object for which you configure remote write: `openshift-monitoring` for default platform monitoring or `openshift-user-workload-monitoring` for user workload monitoring.
```

Reply:
I wonder if we should unpack the meaning a bit to make it easier for non-fluent readers. Perhaps:
```suggestion
The secret must be in the same namespace as the Prometheus object for which you configure remote write. That is, for default platform monitoring, the secret must be in the `openshift-monitoring` namespace. Or, for user workload monitoring, the secret must be in the `openshift-user-workload-monitoring` namespace.
```